tests/queries/0_stateless/02125_fix_storage_filelog.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE log (A String) ENGINE= FileLog('/tmp/aaa.csv', 'CSV'); -- {serverError 36 }
+CREATE TABLE log (A String) ENGINE= FileLog('/tmp/aaa.csv', 'CSV'); -- {serverError 36 }
+CREATE TABLE log (A String) ENGINE= FileLog('/tmp/aaa.csv', 'CSV'); -- {serverError 36 }
```

Comment:
May be in this test after creating file log engine with incorrect path, test creating with correct path (so it would be same as in the issue)?

Reply:
If creating with correct path after creating with incorrect path, it will success.

Reply:
I believe. Ok I see that is not easy to add such test because in will have to be .sh test and .sh test does not allow test hints to suppress expected error. Then ok as it is now.

Reply:
Hm, no, actually it is ok to add one more sh test. 
(I believe that it works, but it might get broken in the future again)

Reply:
OK.